test/web-platform-tests/to-run.yaml

```diff
@@ -850,6 +850,63 @@ DIR: html/webappapis/timers
+extension/pointerevent_constructor.html: [fail, getCoalescedEvents not implemented]
+extension/pointerevent_getCoalescedEvents_when_pointerlocked.html: [timeout, Pointer Lock not unimplemented]
+extension/pointerevent_pointerrawupdate.html: [timeout, Pointer Lock not implemented]
+extension/pointerevent_touch-action-*: [timeout, touch-action not implemented]
+
+idlharness.window.html: [fail, fetch is not defined]
+inheritance.html: [fail, getComputedStyle does not implement initial value of touch-action]
+
+parsing/touch-action-computed.html: [fail, touch-action not implemented]
+#parsing/touch-action-invalid.html: [pass]
+parsing/touch-action-valid.html: [fail, touch-action not implemented]
+
+pointerevent_attributes_hoverable_pointers.html: [timeout, Uses testdriver]
+pointerevent_attributes_nohover_pointers.html: [timeout, Uses testdriver]
+pointerevent_boundary_events_at_implicit_release_hoverable_pointers.html: [timeout, Uses testdriver]
+pointerevent_boundary_events_in_capturing.html: [timeout, Uses testdriver]
+pointerevent_capture_mouse.html: [timeout, Uses testdriver]
+pointerevent_change-touch-action-onpointerdown_touch.html: [timeout, Uses testdriver]
+pointerevent_click_during_capture.html: [timeout, Uses testdriver]
+# pointerevent_constructor.html: [pass]
+pointerevent_disabled_form_control.html: [fail-slow, Uses testdriver]
+pointerevent_element_haspointercapture.html: [fail-slow, Uses testdriver]
+pointerevent_lostpointercapture_for_disconnected_node.html: [fail-slow, Uses testdriver]
+pointerevent_lostpointercapture_is_first.html: [fail-slow, Uses testdriver]
+pointerevent_mouse_capture_change_hover.html: [fail, Uses testdriver]
+#pointerevent_on_event_handlers.html: [pass]
+pointerevent_pointercancel_touch.html: [fail-slow, Uses testdriver]
+pointerevent_pointercapture_in_frame.html: [fail-slow, Uses testdriver]
+pointerevent_pointerenter_does_not_bubble.html: [fail-slow, Uses testdriver]
+pointerevent_pointerleave_after_pointercancel_touch.html: [fail-slow, Uses testdriver]
+pointerevent_pointerleave_descendant_over.html: [fail-slow, Uses testdriver]
+pointerevent_pointerleave_descendants.html: [fail-slow, Uses testdriver]
+pointerevent_pointerleave_does_not_bubble.html: [fail-slow, Uses testdriver]
+pointerevent_pointermove.html: [fail-slow, Uses testdriver]
+pointerevent_pointermove_isprimary_same_as_pointerdown.html: [fail-slow, Uses testdriver]
+pointerevent_pointermove_on_chorded_mouse_button.html: [fail-slow, Uses testdriver]
+pointerevent_pointerout_after_pointercancel_touch.html: [fail-slow, Uses testdriver]
+pointerevent_pointerout_pen.html: [fail-slow, Uses testdriver]
+pointerevent_pointerout_received_once.html: [fail-slow, Uses testdriver]
+pointerevent_releasepointercapture_*: [fail-slow, Pointer Capture not implemented]
+pointerevent_root_computed_style.html: [fail, getComputedStyle does not support inheritance]
```

Comment:
https://github.com/jsdom/jsdom/pull/2668 would enable implementing this since it includes  the ability to define initial values of CSS properties.